client/admin/utils/contentTypeIcons.js

```diff
@@ -0,0 +1,26 @@
+import BookIcon from '@material-ui/icons/Book';
+import ExplicitIcon from '@material-ui/icons/Explicit';
+import AttachmentIcon from '@material-ui/icons/Attachment';
+import TextFieldsIcon from '@material-ui/icons/TextFields';
+import PhoneIcon from '@material-ui/icons/Phone';
+import TvIcon from '@material-ui/icons/Tv';
+import AirplayIcon from '@material-ui/icons/Airplay';
+
```

Comment:
Import icons from '@material-ui/icons' doesn't work 
Import * as icons from '@material-ui/icons' doesn't work because returns a namespace, not an array
we can write icons.TvIcon but can't icons.map or icons[variable] 
What should we do?